identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -69,40 +69,41 @@ <h2 class="form__heading">Newsletter options</h2>
-        <button class="manage-account__modalCloser js-manage-account__modalCloser">
-        @fragments.inlineSvg("cross", "icon")
-        </button>
-        <fieldset class="fieldset">
-            <div class="fieldset__heading">
-                <h2 class="form__heading">Formatting</h2>
-            </div>
 
-            <div class="fieldset__fields">
-                <ul class="u-unstyled">
-
-                    <li class="form-field @if(emailPrefsForm("htmlPreference").errors.nonEmpty) {form-field--error}">
-                        <label class="label">In which format would you like to receive email?</label>
-                        <div class="form-field__note">
-                            HTML emails have formatted text, images and look better. Text emails are quicker to download, but don't contain images or any formatting.
-                            <br />
-                            We recommend HTML emails.
-                        </div>
-
-                        @radioField(emailPrefsForm("htmlPreference"), List("HTML" -> "HTML (images and text)", "Text" -> "Text"))(nonInputFields, messages)
-
-                    </li>
-
-                    <li>
-                        <button type="button" class="manage-account__button save__button js-save-button" data-link-name="Save email preferences">Save</button>
-                    </li>
-                </ul>
-            </div>
-
-        </fieldset>
+    <div class="manage-account__modal manage-account__modal--newsletterFormat">
+        <div class="manage-account__modalContent">
+            <button class="manage-account__modalCloser js-manage-account__modalCloser">
+            @fragments.inlineSvg("cross", "icon")
+            </button>
+            <fieldset class="fieldset">
+                <div class="fieldset__heading">
+                    <h2 class="form__heading">Formatting</h2>
+                </div>
+
+                <div class="fieldset__fields">
+                    <ul class="u-unstyled">
+
+                        <li class="form-field @if(emailPrefsForm("htmlPreference").errors.nonEmpty) {form-field--error}">
+                            <label class="label">In which format would you like to receive email?</label>
+                            <div class="form-field__note">
+                                HTML emails have formatted text, images and look better. Text emails are quicker to download, but don't contain images or any formatting.
+                                <br />
+                                We recommend HTML emails.
+                            </div>
+
+                            @radioField(emailPrefsForm("htmlPreference"), List("HTML" -> "HTML (images and text)", "Text" -> "Text"))(nonInputFields, messages)
+
+                        </li>
+
+                        <li>
+                            <button type="button" class="manage-account__button save__button js-save-button" data-link-name="Save email preferences">Save</button>
```

Comment:
Should this be `[type="submit"]`?

Reply:
yes this is all inherited code but i'd rather leave that for another PR as this one is getting quite hefty. It just hit me this dialog is inaccessible without javascript enabled so that's something to fix on that one too